Library/Formula/rstudio-server.rb

```diff
@@ -0,0 +1,49 @@
+class RstudioServer < Formula
+  homepage "http://www.rstudio.com"
+  url "https://github.com/rstudio/rstudio/archive/v0.98.1103.tar.gz"
+  sha256 "084049aae03cbaaa74a848f491d57cffab5ea67372ece0e256d54e04bd5fc6da"
+
+  depends_on "ant" => :build
+  depends_on "cmake" => :build
+  depends_on "r" => :build
+
+  def install
+    # patch the boost install scripts
+    inreplace "dependencies/common/install-boost", "/opt/rstudio-tools/boost",
+      "#{prefix}/boost"
+    inreplace "dependencies/common/install-boost", "sudo ", ""
+    inreplace "src/cpp/CMakeLists.txt", "/opt/rstudio-tools/boost", "#{prefix}/boost"
+
+    # specific path to R explicitly
+    inreplace "dependencies/common/install-packages", "R CMD build",
+      "#{Formula["r"].opt_bin}/R CMD build"
+
+    # install dependencies
+    Dir.chdir "dependencies/common"
+    system "./install-gwt"
+    system "./install-dictionaries"
+    system "./install-mathjax"
+    system "./install-boost"
+    system "./install-packages"
+    system "./install-pandoc"
+    Dir.chdir "../.."
+
+    # the real body
+    Dir.mkdir "build"
+    Dir.chdir "build"
+    system "cmake", "..",
+      "-DRSTUDIO_TARGET=Server",
+      "-DCMAKE_BUILD_TYPE=Release",
+      "-DBOOST_ROOT=#{prefix}/boost/boost_1_50_0",
```

Comment:
Ok so they have a vendored boost that you're using? If so: that's cool (and good, actually).